poetry/core/vcs/git.py

```diff
@@ -127,8 +127,8 @@ def url(self):  # type: () -> str
             "/" + self.pathname.lstrip(":/"),
         )
 
-    def format(self):
-        return "{}".format(self.url, "#{}".format(self.rev) if self.rev else "",)
+    def format(self):  # type: () -> str
+        return self.url
```

Comment:
Not sure here. Shouldn't it be `return "{}".format(self.url, "#{}{}".format(self.rev) if self.rev else "",)`?

Reply:
Well I just removed the redundant parameter for now; since adding that will start failing tests etc. So, decided to retain the current behaviour (ie. not add the `#` bit). We shold fix this elsewhere since this change has no functional change.

Reply:
Ok, makes sense.